lib/CL/devices/cuda/pocl-cuda.c

```diff
@@ -776,20 +783,41 @@ pocl_cuda_alloc_mem_obj (cl_device_id device, cl_mem mem, void *host_ptr)
     {
       result = cuMemHostAlloc (&p->extra_ptr, mem->size,
                                CU_MEMHOSTREGISTER_DEVICEMAP);
-      CUDA_CHECK (result, "cuMemHostAlloc");
+      if (CUDA_CHECK_ERROR (result, "cuMemHostAlloc"))
+        {
+          p->extra_ptr = NULL;
```

Comment:
Is this after clang-format? The indentation seems to small.

Reply:
Yes, it is after clang-format. The indentation is the same like the whole file: 2 spaces.